### PR TITLE
[X86][MOVRS] Support MOVRS

### PR DESCRIPTION
**Pull request by MalaySanghi**
Ref.: https://cdrdv2.intel.com/v1/dl/getContent/671368

### CODE REVIEW

**Comment by MalaySanghi** [comment]
@phoebewang please review

**Review by phoebewang — APPROVED** [review]
LGTM.

File: .icslock
[the remark this answers is not in the document]
Comment (phoebewang) [reply .icslock:1]: Yes, thanks for pointing out! Unfortunately, I met network problem when committing the fix to GH. @MalaySanghi could you commit a fix instead?

[the remark this answers is not in the document]
Comment (phoebewang) [reply .icslock:1]: The network is recovered, fixed by https://github.com/llvm/llvm-project/commit/9e1faa834173f57344a12b1a0a2f90b8e903c7bd

[the remark this answers is not in the document]
Comment (MalaySanghi) [reply .icslock:1]: Apologies, this escaped my notice. Thanks phoebe for fixing this

**Pull request merged**